src/Illuminate/Contracts/Routing/Registrar.php

```diff
@@ -74,7 +74,7 @@ public function match($methods, $uri, $action);
      * @param  string  $name
      * @param  string  $controller
      * @param  array   $options
-     * @return void
+     * @return \Illuminate\Routing\PendingResourceRegistration
```

Comment:
this PR should target 5.5, because this is a breaking change to the interface, requiring implementations to return this object